tests/modules/biobambam/bamsormadup/test.yml

```diff
@@ -0,0 +1,12 @@
+- name: biobambam/bamsormadup test_biobambam_bamsormadup
+  command: nextflow run tests/modules/biobambam/bamsormadup -entry test_biobambam_bamsormadup -c tests/config/nextflow.config
+  tags:
+    - biobambam/bamsormadup
+    - biobambam
+  files:
+    - path: output/biobambam/test.bam
+      md5sum: 1cf7f957eb20b4ace9f10d0cf0a0649a
+    - path: output/biobambam/test.metrics.txt
+      md5sum: 30d6e7d90bb5df46329d4bc0144ce927
+    - path: output/biobambam/versions.yml
+      md5sum: 0d6f3137ed4515333d73c779f2c24445
```

Comment:
```suggestion
```

Apparently the general policy is not to include the MD5 sum anymore for versions (not yet documented) because makes it annoying when auto-updating the versions of the tools